tests/test_generate_changelog.py

```diff
@@ -0,0 +1,103 @@
+import os
```

Comment:
Please create a directory for the release tool, let's give this thing a name, eg: `releaser` or any other nice name you can think of. We'll concentrate all the test files for the different aspect of this tool in this directory.